### DISCUSSION TITLE
'No Module' error after datapane installed successfully

### DISCUSSION THREAD

**Discussion by charlotter62** [datapane/datapane#86] [Help] [answered]
Hello! I used the command   !pip3 install -U datapane     and it supposedly installed successfully. However it does not recognize the package when I try to import it (import datapane as dp)
Any idea how to fix this?

**Comment by charlotter62** [comment]
I tried         !conda install -c conda-forge "datapane>=0.9.0"       as well and got these issues....

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
                                                                               failed
UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

**Comment by mands** [comment, marked as answer]
Hi @charlotter62 ,

Sorry for the install issues you're hitting, can you provide some more details on your system/environment?

- OS:
- Python version:
- Python environment: pip / conda
- Using jupyter: yes/no

**Reply by charlotter62** [reply]
Thanks for your reply! I am using macOS 10.15.7, activated a conda environment and then launched jupyter notebook. That is where I ran !pip3 install -U datapane and it supposedly installed (datapane --version works, but import datapane does not). I noticed that the python version being used in jupyter notebook is 3.9, where if I check the python version on my terminal it was python 3.74 or python3 3.8. I installed python 3.9 but that may have caused more issues than solved...

**Reply by lanthias** [reply]
Hey Charlotte, is it possible your notebook kernel is running in a different virtual environment than your system python?

**Reply by mands** [reply]
Hi @charlotter62 - yes it seems that the version of Python in your notebook is different from your main OS version of Python, and datapane is being installed into the OS/system version - hence can't be seen by your Jupyter notebook session.

I imagine the issue is that you are running Jupyter in conda, but installing datapane via pip. Could you try installing datapane i the same conda environment as jupyter, i.e. try running `conda install -c conda-forge "datapane>=0.10.0"` to install it and let us know how you get on - thanks!

**Reply by charlotter62** [reply]
Hi again, thanks so much for your feedback. I ended up completely removing anaconda from my computer using CleanMyMac and then reinstalling anaconda navigator, as well as updating python. I think it help reset some of the inconsistencies. Then I installed datapane in the navigator manually under the conda forge channel and had no problems.

**Reply by mands** [reply]
Great to hear!
